### DISCUSSION TITLE
bluecoolman's VCR Recording Preset V4

### DISCUSSION THREAD

**Discussion by oisintheblue** [ntsc-rs/ntsc-rs#113] [Presets]
![image](https://github.com/valadaptive/ntsc-rs/assets/100409599/16e53ab9-d27e-40b0-a9ae-fc9695e1160e)

![image](https://github.com/valadaptive/ntsc-rs/assets/100409599/b01b7e7e-5af7-481a-85de-c0889f271dca)
made my own ntsc-rs preset for you guys to use! 
[bluecoolman_VCRRecordingV4.json](https://github.com/valadaptive/ntsc-rs/files/15474769/bluecoolman_VCRRecordingV4.json)
have fun

**Comment by oisintheblue** [comment]
![image](https://github.com/valadaptive/ntsc-rs/assets/100409599/e5975850-81c0-43fb-8d2a-da4dbbd8e1bc)
oh yeah, almost forgot about the color bars example.

**Reply by smfthe1199** [reply]
Great overall!

**Comment by oisintheblue** [comment]
[bluecoolman_VCRRecordingV5.json](https://github.com/user-attachments/files/16182866/bluecoolman_VCRRecordingV5.json)
UPDATED! 
changes:
bandwidth scale is now 1.00
composite preemphasis is now 0
luma smear is now 0.620

**Reply by smfthe1199** [reply]
Thx

**Comment by oisintheblue** [comment]
![image](https://github.com/user-attachments/assets/d57a3f1a-dc45-4930-a265-85e90f4062f5)
